Add list of things to change

kelloggm opened the pull request.

kelloggm commented:
I looked at your changes, and they look good. I've also added text to the README to address the two points you made in your last message.

mernst commented:
Martin will add qualifiers and uncomment a test (and maybe remove the other test) before pinging Mike again and asking him to review.

kelloggm commented:
@mernst, I've made the changes you requested:
* I added top and bottom qualifiers
* I uncommented and fixed the subtyping test
* I deleted the other test

The tests now run and pass.
